src/Mailer/Message.php

```diff
@@ -868,7 +868,7 @@ public function getHeaders(array $include = []): array
                 'from', 'sender', 'replyTo', 'readReceipt', 'returnPath',
                 'to', 'cc', 'bcc', 'subject',
             ],
-            false
+            ''
```

Comment:
Why is this wrong? It uses `true` above and seems to check for true/false in the code below.

Reply:
Thanks for spotting this. Need to ensure that the array returned by `getHeaders()` contains strings only but this was the wrong fix.